test/CMakeLists.txt

```diff
@@ -8,33 +15,37 @@
 # It also lets running the tests much faster.
 if (APPLE)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ffp-contract=fast")
-endif()  
+endif()
 
 function(boost_geometry_add_unit_test prefix item)
   set(unit_test_name "boost_geometry_${prefix}_${item}")
   add_executable(${unit_test_name} ${item}.cpp)
 
-  # Add a dependendcy to Boost.Geometry
-  target_link_libraries(${unit_test_name} 
+  # Add a dependency to Boost.Geometry
+  target_link_libraries(${unit_test_name}
     PRIVATE
       Boost::geometry)
 
   # For unit tests, add a dependency to the unit test framework (in header only mode)
-  target_link_libraries(${unit_test_name} 
+  target_link_libraries(${unit_test_name}
     PRIVATE
       Boost::included_unit_test_framework)
-  
+
   # Include the main Geometry test folder and the current folder
   target_include_directories(${unit_test_name}
     PRIVATE
-      "${PROJECT_SOURCE_DIR}/test" 
+      "${PROJECT_SOURCE_DIR}/test"
+      "${PROJECT_SOURCE_DIR}/index/test"
       .)
 
   # To compile with C++14
   target_compile_features(${unit_test_name} PRIVATE cxx_std_14)
 
   # To be able to run ctest
-  add_test(NAME ${unit_test_name} COMMAND ${unit_test_name})
+  if (NOT ("boost_geometry_${prefix}_${item}" STREQUAL "boost_geometry_cs_undefined_setops1") AND
+      NOT ("boost_geometry_${prefix}_${item}" STREQUAL "boost_geometry_cs_undefined_setops2"))
```

Comment:
Isn't this too specific?

Reply:
And why is it excluded. I get a memory violation - so that might be the reason.
Can't we pass an optional boolean variable to this function, to add to the test suite, or not?
(and then for ones like this we should not, but with a comment why not).

And then, probably, it can still be added as a unit test - but not as dependency to the custom `tests` target

Reply:
Done, thanks.